conans/client/source.py

```diff
@@ -142,10 +157,11 @@ def _run_source(conanfile, conanfile_path, src_folder, hook_manager, reference,
                 output.info('Configuring sources in %s' % src_folder)
                 _run_scm(conanfile, src_folder, local_sources_path, output, cache=cache)
 
+                get_sources_from_exports()
```

Comment:
I'm passing a function and calling it here to have the same execution order in the local workflow and inside the cache (keeping the execution order that was performed in the cache). I don't like it, and I think that we could run the exports-related copies before running _scm_ ones, what do you think?

Reply:
I don't know if we consider it better like this, but the precedence would change and it would be breaking.

Reply:
Maybe it was a bug, hahaha 🙄